toplevel/topcommon.ml

```diff
@@ -380,3 +380,41 @@ let try_run_directive ppf dir_name pdir_arg =
             dir_name dir_type arg_type;
           false
   end
+
+(* Overriding exception printers with toplevel-specific ones *)
+
+let loading_hint_printer ppf err =
+  Symtable.report_error ppf err;
+  match err with
+  | Symtable.Undefined_global s ->
+    if !Sys.interactive then begin
+      let find_with_ext ext =
+        try Some (Load_path.find_uncap (s ^ ext)) with Not_found -> None
+      in
+      fprintf ppf
+        "@.Hint: @[\
+         This means a module's interface is loaded, \
```

Comment:
I'm not fond of the `'s`. I would suggest: "This means that the interface of a module is loaded,".